shenyu-common/src/test/java/org/apache/shenyu/common/utils/ParamCheckUtilsTest.java

```diff
@@ -28,11 +31,28 @@
 public final class ParamCheckUtilsTest {
 
     @Test
-    public void testDubboBodyIsEmpty() {
-        assertTrue(ParamCheckUtils.dubboBodyIsEmpty(null));
-        assertTrue(ParamCheckUtils.dubboBodyIsEmpty(""));
-        assertTrue(ParamCheckUtils.dubboBodyIsEmpty("{}"));
-        assertTrue(ParamCheckUtils.dubboBodyIsEmpty("null"));
-        assertFalse(ParamCheckUtils.dubboBodyIsEmpty("123"));
+    public void testBodyIsEmpty() {
+        assertTrue(ParamCheckUtils.bodyIsEmpty(null));
+        assertTrue(ParamCheckUtils.bodyIsEmpty(""));
+        assertTrue(ParamCheckUtils.bodyIsEmpty("null"));
+        assertFalse(ParamCheckUtils.bodyIsEmpty("123"));
+    }
+
+    @Test
+    public void testcheckParamsLength() {
+        try {
+            ParamCheckUtils.checkParamsLength(2, 2);
+        } catch (Exception ex) {
+            assertTrue(Objects.isNull(ex));
+        }
+    }
+
+    @Test
+    public void testcheckParamsLengthException() {
+        try {
+            ParamCheckUtils.checkParamsLength(1, 2);
+        } catch (Exception ex) {
+            assertTrue(ex instanceof ShenyuException);
+        }
```

Comment:
`org.junit.jupiter.api.Assertions#assertThrows(java.lang.Class<T>, org.junit.jupiter.api.function.Executable)` will be better.